index.js

```
@@ -31,7 +31,11 @@ export default function(options, storage, key) {
   }
 
   function setState(key, state, storage) {
-    return storage.setItem(key, JSON.stringify(state));
+    try {
+      return storage.setItem(key, JSON.stringify(state));
+    } catch (err) {}
```

Not catching errors is a bad practice... I understand this can be an issue, tho. But ho would the library handle this; @robinvdvleuten?

---

Maybe a `console.log(err)`?

---

It should throw the error and the app should handle it

---

`throw new Error("No set value");`